velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -289,5 +284,56 @@ TEST_F(NthValueTest, emptyFrames) {
   }
 }
 
+// DuckDB has errors in IGNORE NULLS logic when the start or end
+// bound is CURRENT ROW and also when both frame ends are
+// preceding or following. So the testing is limited to this
```

Comment:
The bugs for CURRENT ROW and frames was fixed in DuckDB 0.8.0. I was able to verify that.

At this point we are blocked upgrading DuckDB. 

I feel more confident adding usage of IGNORE NULLS in this test https://github.com/prestodb/presto/pull/19847 and create a github issue on the Velox side to add the tests post upgrade.

What do you think ?

Reply:
Without the tests, this functionality can break easily and it will block advancing velox for Prestissimo. I suggest to add at least some basic tests using hard-coded expected results and add a TODO to add more extensive tests when DuckDB is upgraded. BTW, @majetideepak has a good plan on the upgrade.

Reply:
Done.